docs/source/reference.rst

```diff
@@ -199,6 +199,7 @@ solver options.
 In the 2nd line ``abc`` is the engine, there are no engine options, ``sim3`` is the
 solver, and ``-W 15`` are solver options.
 
+
```

Comment:
What's with the extra blank lines, were those added by the autoformat?

Reply:
This was just mistakenly left in when I ripped the rough docs out, forgot to clear that last line.